src/components/DisciplineItem.tsx

```diff
@@ -1,60 +1,97 @@
-import React, { ReactElement } from 'react'
+import React, { ReactElement, useState } from 'react'
 import { Pressable } from 'react-native'
 import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
 import styled from 'styled-components/native'
 
 import { ChevronRight } from '../../assets/images'
-import { Discipline } from '../constants/endpoints'
 import { COLORS } from '../constants/theme/colors'
 
-const Container = styled(Pressable)<{ selected: boolean }>`
+const Container = styled(Pressable)<{ pressed: boolean }>`
   min-height: ${wp('22%')}px;
   margin: 0px 16px 8px 16px;
   padding: 12px 8px 12px 16px;
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
-  background-color: ${prop => (prop.selected ? prop.theme.colors.lunesBlack : prop.theme.colors.white)};
-  border: 1px solid ${prop => (prop.selected ? prop.theme.colors.lunesBlack : prop.theme.colors.lunesBlackUltralight)};
+  background-color: ${prop => (prop.pressed ? prop.theme.colors.lunesBlack : prop.theme.colors.white)};
+  border: 1px solid ${prop => (prop.pressed ? prop.theme.colors.lunesBlack : prop.theme.colors.lunesBlackUltralight)};
   border-radius: 2px;
 `
-const Title = styled.Text<{ selected: boolean }>`
+const Title = styled.Text<{ pressed: boolean }>`
   font-size: ${props => props.theme.fonts.largeFontSize};
   letter-spacing: ${props => props.theme.fonts.listTitleLetterSpacing};
   margin-bottom: 2px;
   font-family: ${props => props.theme.fonts.contentFontBold};
-  color: ${props => (props.selected ? props.theme.colors.white : props.theme.colors.lunesGreyDark)};
+  color: ${props => (props.pressed ? props.theme.colors.white : props.theme.colors.lunesGreyDark)};
 `
+
 const Icon = styled.Image`
-  justify-content: center;
-  margin-right: 10px;
   width: ${wp('7%')}px;
   height: ${wp('7%')}px;
 `
 
+const IconContainer = styled.View`
+  justify-content: center;
+  margin-right: 10px;
+`
+
 const TextContainer = styled.View`
   flex: 1;
 `
 
-export interface DisciplineItemProps {
-  item: Discipline
-  children: ReactElement
-  selected: boolean
+const DescriptionContainer = styled.View`
+  flex-direction: row;
+  align-items: center;
+`
+
+const Description = styled.Text<{ pressed: boolean }>`
+  font-size: ${props => props.theme.fonts.defaultFontSize};
+  font-weight: ${props => props.theme.fonts.lightFontWeight};
+  font-family: ${props => props.theme.fonts.contentFontRegular};
+  color: ${props => (props.pressed ? props.theme.colors.lunesWhite : props.theme.colors.lunesGreyMedium)};
```

Comment:
The color is now `lunesWhite` (`white` before). Not sure if that is correct but in this screen it is that way: https://app.zeplin.io/project/5fc7c98a435298126bcfd4ee/screen/5fd7ad2c3cfec77f5c68f10c